source/material_model/visco_plastic.cc

```diff
@@ -91,7 +124,6 @@ namespace aspect
     }
 
 
-
```

Comment:
Please add this line again, I think you removed it by accident.

Reply:
I have added that. So it seems 'make indent' doesn't take care of that?